README.md

```diff
@@ -119,7 +117,7 @@ m = Manticore('./path/to/binary')
 def hook(state):
   cpu = state.cpu
   print 'eax', cpu.EAX
-  print cpu.read_int(cpu.SP)
+  print cpu.read_int(cpu.ESP)
```

Comment:
Any reason for updating to `ESP`? As opposed to leaving it or updating to `STACK`.

Reply:
The intention here is stack pointer for 32 bit mode, but x86 actually has a register called `SP` which is the low 16 bits of the stack ptr, so that's what this was actually referencing which is incorrect. I think using native reg name here is better than `STACK` as it's more intuitive.